packages/backend/src/modules/finality/analyzers/StarknetFinalityAnalyzer.ts

```diff
@@ -52,14 +52,14 @@ export class StarknetFinalityAnalyzer extends BaseAnalyzer {
 }
 
 function decodeTransaction(data: string) {
-  const signature = `updateStateKzgDA(uint256[] programOutput, bytes kzgProof)`
+  const signature = `updateStateKzgDA(uint256[] programOutput, bytes[] kzgProofs)`
```

Comment:
this kzgProofs is not used at all, lets remove this field from parsing and logic

Reply:
Removed